dashboard/app/helpers/levels_helper.rb

```diff
@@ -303,7 +303,7 @@ def blockly_options(local_assigns={})
       },
       droplet: @game.try(:uses_droplet?),
       pretty: Rails.configuration.pretty_apps ? '' : '.min',
-      applabUserId: @applab_user_id,
+      applabUserId: applab_user_id,
```

Comment:
If you don't want to add this property to non-applab levels you could do the following:
```ruby
app_options[:applabUserId] = applab_user_id if @game == Game.applab
```